src/index.html

```diff
@@ -81,6 +81,14 @@
       enabled browser listed at <a href="http://www.webrtc.org">http://www.webrtc.org</a>.</p>
   </paper-dialog>
 
+  <!-- GetUserMedia no audio or video device dialog -->
+  <paper-dialog id="gum-no-device-dialog"
+                heading="Welcome to WebRTC Troubleshooter"
+                backdrop autoCloseDisabled>
+    <p>Your device does not have any audio or video devices connected.
+    Please, plug a microphone or a camera into your device.</p>
```

Comment:
Indent by 2 spaces.

I know there is a place above where it''s indented incorrectly (most likely by me;)), will fixed that part of a different PL.